Small changelog updates (#4353)
zeroc-ice/ice: CHANGELOG-3.8.md

## Ruby Changes

> - There is no Ruby-specific update in this release. Nevertheless, we made many updates to Ice for Ruby: see
>   [General Changes](#general-changes) and [Slice Language Changes](#slice-language-changes).

Reviewer: The tense sounds a little off here, I think it should be:
```suggestion
- There are no Ruby-specific updates in this release. Nevertheless, we made many updates to Ice for Ruby: see
  [General Changes](#general-changes) and [Slice Language Changes](#slice-language-changes).
```
